chart/compass/templates/system-fetcher-job.yaml

```diff
@@ -25,7 +25,7 @@ spec:
               hostnames:
                 # Used for calls to Director
                 - "{{ .Values.global.gateway.tls.secure.internal.host }}.{{ .Values.global.ingress.domainName }}"
-                - "{{ .Values.global.externalServicesMock.certSecuredHost }}.{{ .Values.global.ingress.domainName }}"
+                - "{{ .Values.global.externalServicesMock.ordCertSecuredHost }}.{{ .Values.global.ingress.domainName }}"
```

Comment:
This needs to be certSecuredHost as the system fetcher don't do anything ORD related.